app/src/main/kotlin/com/wire/android/ui/authentication/create/username/CreateAccountUsernameViewModel.kt

```diff
@@ -11,9 +11,7 @@ import com.wire.android.navigation.BackStackMode
 import com.wire.android.navigation.NavigationCommand
 import com.wire.android.navigation.NavigationItem
 import com.wire.android.navigation.NavigationManager
-import com.wire.android.util.USERNAME_FORBIDDEN_CHARACTERS_REGEX
-import com.wire.android.util.USERNAME_MAX_LENGTH
-import com.wire.android.util.USERNAME_MIN_LENGTH
```

Comment:
I think we can remove `UsernameUtil` completely as we use only `ValidateUserHandleUseCase`

Reply:
I did delete it 👍🏼